src/Configuration/src/CloudFoundry/CloudFoundryServiceCollectionExtensions.cs

```diff
@@ -14,17 +14,17 @@ namespace Steeltoe.Extensions.Configuration.CloudFoundry;
 public static class CloudFoundryServiceCollectionExtensions
 {
     /// <summary>
-    /// Bind configuration data into <see cref="CloudFoundryApplicationOptions" /> and <see cref="CloudFoundryServicesOptions" /> and add both to the
+    /// Binds configuration data into <see cref="CloudFoundryApplicationOptions" /> and <see cref="CloudFoundryServicesOptions" /> and adds both to the
     /// provided service container as configured TOptions.  You can then inject both options using the normal Options pattern.
```

Comment:
I think TOption is correct, see line 67

Reply:
I've removed that part the the doc, because this method doesn't have a TOptions type parameter. Should it have one? Currently its hardcoded to `CloudFoundryApplicationOptions`.